test/Base_Tests/src/Semantic/Any_Spec.enso

```diff
@@ -38,75 +38,79 @@ add_specs suite_builder =
             (1 != 1) . should_be_false
 
     suite_builder.group "Any's methods" group_builder->
-        group_builder.specify "should not be overridable when called statically" <|
+        group_builder.specify "Instance method invocation" <|
```

Comment:
This diff is more important than https://github.com/enso-org/enso/pull/13961#issuecomment-3366485097 and it’s https://github.com/enso-org/enso/commit/7b589378a49dbcb104f40166c064526a31ece732 as it shows the diff against current behavior. Can we make it *less red*?